docs/tutorials/basic/module.md

```diff
@@ -141,15 +164,27 @@ mod.fit(train_iter,
 
 ## Intermediate-level Interface
 
-We already saw how to use module for basic training and inference. Now we are
-going to see a more flexible usage of module. Instead of calling
-the high-level `fit` and `predict` APIs, one can write a training program with the intermediate-level APIs
-`forward` and `backward`.
+We can also use module's intermediate-level APIs to do training and inference.
```

Comment:
I tried doing it. But to run fit function, we need to create iterator first. If we describe intermediate level API first, we won't have any remaining batch in train_iter as we loop through it. So, we will again need to create the iterator to feed into fit function.

Will add fit API docs link.